src/scripts/extensions/webExtensionBase/webExtensionWorker.ts

```diff
@@ -71,7 +70,7 @@ export class WebExtensionWorker extends ExtensionWorkerBase<W3CTab, number> {
 	protected doSignOutAction(authType: AuthType) {
 		let usidQueryParamValue = this.getUserSessionIdQueryParamValue();
 		let signOutUrl = ClipperUrls.generateSignOutUrl(this.clientInfo.get().clipperId, usidQueryParamValue, AuthType[authType]);
-		BrowserUtils.appendHiddenIframeToDocument(signOutUrl);
+		fetch(signOutUrl);
```

Comment:
For my information - why are we doing a fetch call here? I believe the previous code fragment was adding a hidden iframe

Reply:
Yes, the previous code was adding a hidden iframe using DOM elements which won't be possible now. I explored using the same function which is used for sign in, i.e. `launchWebExtensionPopupAndWaitForClose`. However, it would result in the creation of an additional sign out screen, just as we get an additional screen while signing in, and we would no longer have a 1-click sign out experience. Directly fetching the sign out url instead resolves this.